monad-eth-testutil/examples/txgen/run.rs

```diff
@@ -706,9 +713,20 @@ async fn load_or_deploy_contracts(
             )
             .await?;
 
+            let paymaster = match use_paymaster {
+                true => {
+                    Paymaster::deploy(&deployer, client, &entrypoint, max_fee_per_gas, chain_id)
+                        .await?
+                }
+                false => Paymaster {
+                    addr: Address::ZERO,
+                },
+            };
+
             let erc4337_7702 = ERC4337_7702Bundled {
                 entrypoint: entrypoint.addr,
                 simple7702account: simple7702account.addr,
+                paymaster: paymaster.addr,
```

Comment:
nit: this should probably be an Option<Address> and then have the match on use. Fine to merge as it is though